apps/CrazyCattle3D/install

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# use unofficial source to download CrazyCattle3D resources
```

Comment:
I wouldn't call this an unofficial source. It is from the developers on their own domain after all.